Remotion/Web/Development.WebTesting/RequestErrorDetectionStrategies/AspNetRequestErrorDetectionParser.cs

```diff
@@ -27,6 +32,16 @@ namespace Remotion.Web.Development.WebTesting.RequestErrorDetectionStrategies
   /// </summary>
   public sealed class AspNetRequestErrorDetectionParser
   {
+    [DataContract]
```

Comment:
Please move the ErrorObject after the public nested type to follow visibility practices.